mico-core/src/main/java/io/github/ust/mico/core/model/KFConnectorDeploymentInfo.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.github.ust.mico.core.model;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import io.github.ust.mico.core.dto.request.KFConnectorDeploymentInfoRequestDTO;
+import lombok.*;
+import lombok.experimental.Accessors;
+import org.neo4j.ogm.annotation.*;
+
+/**
+ * Represents an instance of a {@link MicoServiceDeploymentInfo}
+ * for the deployment of a KafkaFaasConnector.
+ * <p>
+ * An instance of this class is persisted as a relationship between
+ * between a {@link MicoApplication} and a {@link MicoServiceDeploymentInfo} node
+ * in the Neo4j database.
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Accessors(chain = true)
+@RelationshipEntity(type = "PROVIDES_KF_CONNECTOR")
```

Comment:
What is the reasoning behind this decision?

Reply:
Do you mean to name the relationship `PROVIDES_KF_CONNECTOR` instead of `PROVIDES`?
Reason: To be able to differentiate in a Neo4j query between a normal `MicoServiceDeploymentInfo` and a `MicoServiceDeploymentInfo` that is used for a KafkaFaasConnector instance.